app/component/SummaryPage.js

```diff
@@ -1856,6 +1875,18 @@ class SummaryPage extends React.Component {
     }
     const onlyHasWalkingItineraries = this.onlyHasWalkingItineraries();
 
+    const itineraryContainsBikeRentalStation = filteredItineraries[
+      activeIndex
+    ]?.legs.some(leg => leg.from.bikeRentalStation);
+
+    const mapLayerOptions = itineraryContainsBikeRentalStation
+      ? this.addBikeStationMapForRentalBikeItineraries(filteredItineraries)
+      : this.props.mapLayerOptions;
+
+    const stopsToHide = this.getListOfHiddenStops(
```

Comment:
Maybe instead of calling them stops we could be a bit more generic and call them `objectsToHide`. Then it should probably be of structure:
```
{
    vehicleRentalStations: [<list of ids>]
}
```
or alternatively just call it `vehicleRentalStationsToHide` but I'm not sure if it's better to have one prop with a mystery structure or multiple props :D.